utility/src/test/java/org/oppia/android/util/logging/performancemetrics/PerformanceMetricsConfigurationsModuleTest.kt

```diff
@@ -0,0 +1,121 @@
+package org.oppia.android.util.logging.performancemetrics
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.Binds
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.testing.robolectric.RobolectricModule
+import org.oppia.android.testing.threading.TestDispatcherModule
+import org.oppia.android.util.data.DataProvidersInjector
+import org.oppia.android.util.data.DataProvidersInjectorProvider
+import org.oppia.android.util.logging.LoggerModule
+import org.oppia.android.util.logging.firebase.LogReportingModule
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+private const val ONE_GIGABYTES = 1024L * 1024L * 1024L
+
+/** Tests for [PerformanceMetricsConfigurationsModule]. */
+// FunctionName: test names are conventionally named with underscores.
+@Suppress("FunctionName")
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(application = PerformanceMetricsConfigurationsModuleTest.TestApplication::class)
+class PerformanceMetricsConfigurationsModuleTest {
+
+  @JvmField
+  @field:[Inject MediumMemoryTierUpperBound]
+  var mediumMemoryTierUpperBound: Long = 0L
+
+  @JvmField
+  @field:[Inject LowMemoryTierUpperBound]
+  var lowMemoryTierUpperBound: Long = 0L
+
+  @JvmField
+  @field:[Inject LowStorageTierUpperBound]
+  var lowStorageTierUpperBound: Long = 0L
+
+  @JvmField
+  @field:[Inject MediumStorageTierUpperBound]
+  var mediumStorageTierUpperBound: Long = 0L
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
+  fun testConfigModule_verifyLowStorageTierUpperBound() {
```

Comment:
You could probably be specific in the test name here, e.g.:

``testLowStorageTierUpperBound_isThirtyTwoGigabytes``

Ditto for the others.

Reply:
done.